lib/app.rb

```diff
@@ -1,17 +1,15 @@
-require_relative 'classroom'
 require_relative 'person'
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'book'
 require_relative 'capitalize'
 require_relative 'trimcase'
 require_relative 'rental'
-
+require 'json'
 class App
   def initialize
-    @books = []
-    @person = []
-    @rentals = []
+    load_books
+    load_persons
   end
```

Comment:
- Great job for implementing book and person data to JSON, I think you forgot to initialize the rental witch leads as error to save, please fix this issue to fit the requirement 👍